app/src/main/java/com/eveningoutpost/dexdrip/Models/AlertType.java

```diff
@@ -361,10 +358,10 @@ public static void update_alert(
         at.save();
     }
     public static void remove_alert(String uuid) {
-  /*      if(uuid.equals(LOW_ALERT_55)) {
+        if(uuid.equals(LOW_ALERT_55)) {
```

Comment:
There should not be any alerts which cannot be removed/disabled if the user wishes to do so. This restriction should be reverted.